software/base/src/main/java/org/apache/brooklyn/entity/machine/MachineInitTasks.java

```diff
@@ -75,13 +72,19 @@ public Void call() {
     /**
      * Returns a queued {@link Task} which stops iptables on the given machine.
      */
-    public Task<Void> stopIptablesAsync(final SshMachineLocation machine) {
-        return DynamicTasks.queue("stop iptables", new Callable<Void>() {
-            public Void call() {
-                stopIptablesImpl(machine);
-                return null;
-            }
-        });
+    public ProcessTaskWrapper<Integer> stopIptablesAsync(final SshMachineLocation machine) {
```

Comment:
Change all methods to return `Task` instead of `ProcessTaskWrapper`, use `.asTask()` to get the task from the `queue` return value.